views/templates/front/orderDetail.tpl

```diff
@@ -23,6 +23,11 @@
  *}
  
  <section class="box" id="order-detail">
+
+    {if isset($token_in_url) && $token_in_url === true}
```

Comment:
=== true is facultative

Reply:
token_in_url has the "false" value by default in the hookDisplayOrderDetail, i think if we remove the === true condition, we would go into the "if" when we don't want to ?

Reply:
ok